docs/src/conf.py

```diff
@@ -210,6 +214,7 @@
     ("py:class", "xml.dom.minidom.Element"),
     ("py:func", "django.contrib.sitemaps.Sitemap._urls"),
     ("py:func", "django.utils.text.capfirst"),
+    ("py:class", "import_export.resources.Resource.get_instance"),
```

Comment:
```suggestion
```
This doesn't work. You can only exclude entities that don't have documentation, you cannot ignore errors in existing docs here.